packages/@okta/vuepress-site/docs/reference/api/event-hooks/index.md

```diff
@@ -33,6 +33,8 @@ Note that the API key you set here is unrelated to the Okta API token you must s
 
 You can also optionally specify extra headers that you wish Okta to pass to your external service with each call.
 
+Your external service's endpoint needs to be an HTTPS endpoint, and therefore the URI you specify should always begin with `https://`.
```

Comment:
nit : I'd suggest saying `...needs to be a valid HTTPS endpoint..` to make it clear that fake HTTPS endpoints do not work